### DISCUSSION TITLE
XMLHttpRequest.upload events with msw@next

### DISCUSSION THREAD

**Discussion by receptiryaki** [mswjs/msw#1670] [Q&A]
Hello everyone

I'm trying to make a file upload progress with msw@next (0.0.0-fetch.rc-15)

Simply I used MDN documentation example for the client.
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequestUpload

But events not working correctly.

Here is the codesandbox example.
https://codesandbox.io/s/headless-bush-w5vrks

Just I'm trying to understand if that possible with MSW.

Thanks for the help.

**Comment by kettanaito** [comment]
Hi, @receptiryaki. Thanks for opening this. 

Yes, uploading data should be possible with MSW. If it isn't, it's a bug and we will look into it. 

Thanks for providing the sandbox! I will try to check what's going on next week.
